app/models/weight_log.rb

```diff
@@ -0,0 +1,9 @@
+class WeightLog < ApplicationRecord
+  validates_presence_of :weight
+
+  belongs_to :client
+
+  def self.latest
+    order(created_at: :desc).take
```

Comment:
```suggestion
    where("user_id = current_user.id").order(created_at: :desc).take
```
Should be scoped to current user, otherwise returns the latest log in the entire system